core/render/__init__.py

```diff
@@ -4,16 +4,14 @@
 from . import json
 from . import vi
 from os import path
-
+from viur.core import conf
 
 def isViAvailable():
-    from viur.core.utils import projectBasePath
-    return path.exists(path.join(projectBasePath, "vi", "main.html"))
+    return path.exists(path.join(conf["viur.instance.core_base_path"], "vi", "main.html"))
 
 
 def isAdminAvailable():
```

Comment:
General question: Is this only a relict of the Pyodie-based QT-Admin-Test?

Reply:
No it`s used here https://github.com/viur-framework/viur-core/blob/221743136e867911bfa0fa994ed9db4ae820609d/core/render/vi/__init__.py#L152-L155 in example.

Reply:
Nevermind. We should kill this later on.